src/main/webapp/js/ideas/ideaUpdate.controller.js

```diff
@@ -5,13 +5,34 @@
         .module('app.controllers')
         .controller('updateIdea', updateIdea);
 
-    updateIdea.$inject = ['$scope', '$window', 'restFactory', 'ideasFactory', 'detailsService', 'ideaDetails'];
+    updateIdea.$inject = ['$scope', '$window', /*'restFactory', */'ideasFactory', 'detailsService', 'ideaDetails'];
 
-    function updateIdea($scope, $window, restFactory, ideasFactory, detailsService, ideaDetails) {
+    function updateIdea($scope, $window, /*restFactory, */ideasFactory, detailsService, ideaDetails) {
 
         this.categories =  detailsService.getCategories();
         $scope.bottomButtonName = 'Обновить';
-        $scope.data = ideaDetails;
+        $scope.idea = ideaDetails;
+        $scope.data = null;
+
+        //maps data
+		$scope.center = [30.331014, 53.894617];
+		var map = null;
+		var ideaCoords = null;
+
+        this.promises = ideasFactory.getIdeaById($scope.idea.id).then(
+                                       //success
+                                       function( value )
+                                       {
+                                        $scope.data = value;
+
+                                        //set geo point
```

Comment:
We use 2 different way for display map (original Yandex map and ya-map directive). As for me, it's make sense only for ya-map directive. I'll do this now.